### PR TITLE
CA-8054 : Lobbyist Ruby : Backend Changes for uploading new Images

### PR DESCRIPTION
**Pull request by tp-beesolver**

### CODE REVIEW

File: lib/lobbyist/v2/zipped_image_upload.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+module Lobbyist
+  module V2
+    # API wrapper for v2/zipped_image_uploads
+    class ZippedImageUpload < Lobbyist::V2::Base
+      attr_accessor :id, :name, :filename, :row_count, :imported_count
+      attr_accessor :processed_at, :created_at, :updated_at
+
+      def self.create(params = {})
+        create_from_response(post('/v2/zipped_image_uploads.json', { 'zipped_image_upload' => params }, true))
```
Comment (sachinswagh-opcito) [inline lib/lobbyist/v2/zipped_image_upload.rb]: Prefer symbol over single quoted string and {k1: params} syntax for key in { 'zipped_image_upload' => params }

File: lib/lobbyist/v2/zipped_image_upload.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+      end
+
+      def self.find(id)
+        create_from_response(get("/v2/zipped_image_uploads/#{id}.json"))
+      end
+
+      def self.update(id, params = {})
+        create_from_response(put("/v2/zipped_image_uploads/#{id}.json", 'zipped_image_upload' => params))
```
Comment (sachinswagh-opcito) [inline lib/lobbyist/v2/zipped_image_upload.rb]: Prefer symbol over single quoted string and {k1: params} syntax for key in { 'zipped_image_upload' => params }

File: lib/lobbyist/v2/zipped_image_upload.rb (hunk 1 above)
Comment (tp-beesolver) [reply lib/lobbyist/v2/zipped_image_upload.rb]: @sachinswagh-opcito Done

File: lib/lobbyist/v2/zipped_image_upload.rb (hunk 2 above)
Comment (tp-beesolver) [reply lib/lobbyist/v2/zipped_image_upload.rb]: @sachinswagh-opcito Done

File: Gemfile.lock [hunk 3]
```diff
@@ -1,7 +1,7 @@
 PATH
   remote: .
   specs:
-    lobbyist-ruby (0.7.83)
+    lobbyist-ruby (0.7.84)
```
Comment (sachinswagh-opcito) [inline Gemfile.lock]: Please check this version

Comment (tp-beesolver) [reply Gemfile.lock]: @sachinswagh-opcito Updated.

**Pull request merged**